src/main/java/core/basesyntax/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class DataConverterImpl implements DataConverter {
+    private static final String COMMA_SEPARATOR = ",";
+    private static final int EXPECTED_COLUMNS = 3;
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+    private static final int HEADER_ROW_INDEX = 0;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> transactionFile) {
+        return IntStream.range(1, transactionFile.size())
```

Comment:
Let’s follow the KISS principle. 
Make your solution as simple as possible. Don't use a call chain when it complicates your solution.
A good method shouldn't exceed 15-20 lines